src/pushi/base/mail.py

```
@@ -28,8 +28,291 @@
+    The handler requires SMTP configuration via environment variables
+    to function. If SMTP_HOST is not configured, the handler will
+    gracefully skip sending emails.
+
+    :see: https://en.wikipedia.org/wiki/Simple_Mail_Transfer_Protocol
+    """
+
+    def __init__(self, owner):
+        handler.Handler.__init__(self, owner, name="mail")
+        self.subs = {}
+
+    def send(self, app_id, event, json_d, invalid={}):
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)